tests/deployment_utils.py

```diff
@@ -220,18 +220,17 @@ def _create_temp_config(self, config: str):
             f.write(config)
         return file_path
 
-    def run(self, config: str, force_installed: bool = False, vcs_type: str = "none",
-            additional_parameters="", environment=None, expected_to_fail=False, workdir=None):
+    def run(self, config: str, vcs_type: str = "none", force_installed=False, workdir=None,
```

Comment:
Could you, please, change the order of parameters back?
This will minimize the amount of changes on diff.